### DISCUSSION TITLE
誰有推薦的長篇小說可以讓我在寒假閱讀

### DISCUSSION THREAD

**Discussion by yuchien-lin** [1101-GS4719-C/Python-Community-GS4719-C#1866] [General]
寒假快到了，我想利用寒假的時間來看一些劇情比較長的科幻小說，以前看過波西傑克森、混血營英雄，又或是類似金庸那類的武俠小說，我覺得這幾種類型的小說非常好看，彷彿看一看能帶我進入到書裡描寫的世界裡，有人有看過這些類型的小說能推薦嗎?

**Comment by FakeWood** [comment]
我覺得倚天屠龍記很好看，如果日系的話，我推薦魔法科高中的劣等生，我覺得有些輕小說會有一種作者在自嗨的感覺(個人感受，拜託不要噴我QQ)，但這套不會，一樣是主角龍傲天類型的小說，但我覺得看著很舒服。
歐美的話，小時候有看過一套「皇家騎士」，不知道出到哪了，我覺得很好看
另外推一套不太屬於你說的那種類型的小說，「碟型世界特警隊」我覺得很好看，名字很奇怪沒錯，但風格我很喜歡

**Comment by thz0000** [comment]
武俠小說的話金庸全套都很好看！！
看完金庸還可以看古龍的，也是推薦全套~
還有鄭丰的，推薦<靈劍>和<天觀雙俠>，我只看過這兩部，但是好看ww
我真的超喜歡武俠小說的，國小就開始看金庸
國中開始都沒在上課，都在底下偷偷看小說XD
